app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+class Product < ApplicationRecord
+  validates_presence_of :title
```

Comment:
this can be refactored to:
`validates_presence_of :title, :description, :price`

Reply:
done